src/services/datahub-client.ts

```diff
@@ -199,23 +203,25 @@ export class HttpDatahubClient implements DatahubClient {
     }
   }
 
-  public async getDailyVisitsByPeriod({
+  public async getDailyAndHoursVisitsByPeriod({
     domainName,
     dateFrom,
+    periodByhours,
   }: {
     domainName: string;
     dateFrom: Date;
-  }): Promise<DailyVisitsResult> {
+    periodByhours: boolean;
+  }): Promise<DailyAndHoursVisitsResult> {
     try {
       const response = await this.customerGet<any>(
         `domains/${domainName}/events/quantity-summarized-by-period`,
         {
           startDate: dateFrom.toISOString(),
-          periodBy: 'days',
+          periodBy: periodByhours ? 'hours' : 'days',
         },
       );
 
-      const dailyVisits = response.data.periods.map((x: any) => ({
+      const visits = response.data.periods.map((x: any) => ({
```

Comment:
What do you think about:

```suggestion
      const visitsByPeriod = response.data.periods.map((x: any) => ({
```

Or

```suggestion
      const periodVisits = response.data.periods.map((x: any) => ({
```

Reply:
fixed